diff -p -up silc-toolkit-1.1.10/lib/Makefile.am~ silc-toolkit-1.1.10/lib/Makefile.am
--- silc-toolkit-1.1.10/lib/Makefile.am~	2009-09-17 10:45:39.000000000 +0200
+++ silc-toolkit-1.1.10/lib/Makefile.am	2009-09-17 11:00:32.000000000 +0200
@@ -48,8 +48,8 @@ SILC_LINK_LIBS=$(LIBS)
 SILCCLIENT_LINK_LIBS=$(LIBS) -lsilc
 SILCSERVER_LIBS=$(LIBS) -lsilc
 else
-SILC_LINK_LIBS=
-SILCCLIENT_LINK_LIBS=
+SILC_LINK_LIBS=$(LIBS)
+SILCCLIENT_LINK_LIBS=$(top_builddir)/lib/libsilc.la
 SILCSERVER_LIBS=
 endif
 
